tests/test_copydb.py

```diff
@@ -4,16 +4,18 @@
 import os
 import shutil
 import subprocess
+from collections import defaultdict
 
 import pytest
 from click.testing import CliRunner
-from click_odoo import odoo
+from click_odoo import OdooEnvironment, odoo
 
 from click_odoo_contrib._dbutils import db_exists
 from click_odoo_contrib.copydb import main
 
 TEST_DBNAME = "click-odoo-contrib-testcopydb"
 TEST_DBNAME_NEW = "click-odoo-contrib-testcopydb-new"
+_DEFAULT_IR_CONFIG_PARAMETERS = ["database.uuid", "database.create_date"]
```

Comment:
Do we want to handle the other database keys?

```
 database.create_date
 database.secret
 database.uuid
 database.enterprise_code
 database.expiration_date
 database.expiration_reason
```

Reply:
This list is not the same into all the odoo versions... 
I maybe should reuse the definition provided by odoo:
https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/ir_config_parameter.py#L18
```python
_default_parameters = {
    "database.secret": lambda: str(uuid.uuid4()),
    "database.uuid": lambda: str(uuid.uuid1()),
    "database.create_date": fields.Datetime.now,
    "web.base.url": lambda: "http://localhost:%s" % config.get('http_port'),
    "base.login_cooldown_after": lambda: 10,
    "base.login_cooldown_duration": lambda: 60,
}
```
https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/ir_config_parameter.py#L18
```python
_default_parameters = {
    "database.secret": lambda: pycompat.text_type(uuid.uuid4()),
    "database.uuid": lambda: pycompat.text_type(uuid.uuid1()),
    "database.create_date": fields.Datetime.now,
    "web.base.url": lambda: "http://localhost:%s" % config.get('http_port'),
    "base.login_cooldown_after": lambda: 10,
    "base.login_cooldown_duration": lambda: 60,
}
```
https://github.com/odoo/odoo/blob/11.0/odoo/addons/base/ir/ir_config_parameter.py#L18
```python
_default_parameters = {
    "database.secret": lambda: pycompat.text_type(uuid.uuid4()),
    "database.uuid": lambda: pycompat.text_type(uuid.uuid1()),
    "database.create_date": fields.Datetime.now,
    "web.base.url": lambda: "http://localhost:%s" % config.get('http_port'),
}
```
https://github.com/odoo/odoo/blob/10.0/odoo/addons/base/ir/ir_config_parameter.py#L16
```python
_default_parameters = {
    "database.secret": lambda: (str(uuid.uuid4()), ['base.group_erp_manager']),
    "database.uuid": lambda: (str(uuid.uuid1()), []),
    "database.create_date": lambda: (fields.Datetime.now(), ['base.group_user']),
    "web.base.url": lambda: ("http://localhost:%s" % config.get('xmlrpc_port'), []),
}
```
https://github.com/odoo/odoo/blob/9.0/openerp/addons/base/ir/ir_config_parameter.py#L20
```python
_default_parameters = {
    "database.secret": lambda: (str(uuid.uuid4()), ['base.group_erp_manager']),
    "database.uuid": lambda: (str(uuid.uuid1()), []),
    "database.create_date": lambda: (datetime.datetime.now().strftime(misc.DEFAULT_SERVER_DATETIME_FORMAT), ['base.group_user']),
    "web.base.url": lambda: ("http://localhost:%s" % config.get('xmlrpc_port'), []),
}
```
https://github.com/odoo/odoo/blob/8.0/openerp/addons/base/ir/ir_config_parameter.py#L38
```python
_default_parameters = {
    "database.uuid": lambda: (str(uuid.uuid1()), []),
    "database.create_date": lambda: (datetime.datetime.now().strftime(misc.DEFAULT_SERVER_DATETIME_FORMAT), ['base.group_user']),
    "web.base.url": lambda: ("http://localhost:%s" % config.get('xmlrpc_port'), []),
}
```

Reply:
Bu default `database.enterprise_code`,  `database.expiration_date` and  `database.expiration_reason` are never reset. We should maybe do it.